content/docs/guide/map_submission/map_porting.md

```diff
@@ -10,202 +10,133 @@ weight: 2
-
-### Drop Teleports
-
-Some maps, most notably surf maps, teleport the player into a floating "cage" that is open on the bottom to reset the player's velocity after they fail or transition between stages.
-
-![Drop Teleport](/images/map_porting/drop_tele.png)
-
-Momentum Mod introduces the "Keep Negative Z Velocity Only" velocity mode option for `trigger_teleport` that removes any horizontal velocity or upward vertical velocity from entities when they are teleported. This is more robust than only relying on the cage, but more importantly, this activates some important behaviors in Momentum Mod:
-
-- When teleporting directly into a stage zone, it keeps the zone from activating until the player lands on the ground. This results in more competitively correct splits.
-- It prevents players from using air acceleration until they land on the ground, preventing an unfavorable technique to gain speed early.
-
-{{< hint info >}} These behaviors only activate when there is standable ground below the teleport destination, but you should still use "Keep Negative Z Velocity Only" for caged teleport destinations that are not above standable ground. {{< /hint >}}
-
-All teleports with caged teleport destinations should use this velocity mode. In the in-game Entity Tools, open the "Teleport Velocity Mode" dropdown, select the destination that you want to change to a drop teleport, and then select the "Keep Negative Z" radio button.
-
-![Velocity Mode Tools](/images/map_porting/velocity_mode.png)
-
-### Jail Timers
-
-Some old surf maps use a `logic_timer` to teleport all players to a jail after a few minutes. These timer entities and the associated teleport triggers should be removed using Lumper:
+This guide gives an overview of all the steps required to officially port a map to Momentum Mod.  
+The goal of porting is to make minimal changes necessary for the map to function.
+
+You are free to port any publically available map with some exceptions:
+- Mappers can reserve/block porting of their maps using [this form](https://docs.google.com/forms/d/e/1FAIpQLSeheNDY5A960u6GtXCHtt3s_2vZJL3o5tMJ_ZNbYOpb6cx5nQ/viewform).
+- Before porting make sure the map isn't on [this spreadsheet](https://docs.google.com/spreadsheets/d/1KHeWfhGUNpN267CXtPvVdf2h7eQbjPUhWVkE5NimYhg/edit?gid=2051215588#gid=2051215588).
+- When in doubt, contact the mapper ( please don't spam them )
+
+# Setup
+If you have any questions feel free to ask for help in #map-porting channel on our [Discord](https://discord.gg/momentummod)!
+
+1. Download [Lumper](https://github.com/momentum-mod/lumper), we will use it to modify the map
+2. Download the map you want to port (maps in **.bz2** format can be extracted using [7zip](https://www.7-zip.org/)):
```

Comment:
You can add a few more links here:
CS:GO KZ: https://files.femboy.kz/fastdl/csgo/maps/
HLKZ/Bhop: https://hlkz.sourceruns.org/maps (link to goldsrc porting guide)
Defrag: https://ws.q3df.org/ (link to Quake 3 porting guide/BSP convert)